upgrade_test.py

```diff
@@ -891,6 +911,74 @@ def test_kubernetes_scylla_upgrade(self):
                                         'entire test, actual: %d' % (
                 error_factor, schema_load_error_num)
 
+    def _get_current_operator_image_tag(self):
+        return self.k8s_cluster.kubectl(
+            "get deployment scylla-operator -o custom-columns=:..image --no-headers",
+            namespace=self.k8s_cluster._scylla_operator_namespace  # pylint: disable=protected-access
+        ).stdout.strip().split(":")[-1]
+
+    def test_kubernetes_operator_upgrade(self):
+        self.log.info('Step1 - Populate DB with data')
+        self.prepare_keyspaces_and_tables()
+        self.fill_and_verify_db_data('', pre_fill=True)
+
+        self.log.info('Step2 - Run c-s write workload')
+        self.verify_stress_thread(self.run_stress_thread(
+            stress_cmd=self._cs_add_node_flag(self.params.get('stress_cmd_w'))))
+
+        self.log.info('Step3 - Run c-s read workload')
+        self.verify_stress_thread(self.run_stress_thread(
+            stress_cmd=self._cs_add_node_flag(self.params.get('stress_cmd_r'))))
+
+        self.log.info('Step4 - Upgrade scylla-operator')
+        base_docker_image_tag = self._get_current_operator_image_tag()
+        upgrade_docker_image = self.params.get('k8s_scylla_operator_upgrade_docker_image') or ''
```

Comment:
does test have any sense when k8s_scylla_operator_upgrade_docker_image is empty?

Reply:
Value from helm chart will be used in case that option is empty.

Reply:
Let's address it to @bentsi , not sure it is valid case

Reply:
It is valid case because we are interested in helm chart version, it picks up image it needs by itself.